innovation_sweet_spots/getters/inputs.py

```diff
@@ -0,0 +1,48 @@
+"""
+Modules for collecting data analysis inputs.
+
+To fetch all inputs, simply run "python inputs.py"
+"""
+from innovation_sweet_spots import PROJECT_DIR, config, logging
+from pathlib import Path
+from data_getters.gtr import build_projects
+import pandas as pd
+
+gtr_path = f"{PROJECT_DIR}/inputs/data/gtr_projects.csv"
```

Comment:
PEP guidance is to capitalise globals

```suggestion
GTR_PATH = f"{PROJECT_DIR}/inputs/data/gtr_projects.csv"
```